product/src/it/java/com/yas/product/controller/ProductControllerIT.java

```diff
@@ -0,0 +1,526 @@
+            "Product specifications including dimensions, weight, and materials.",
+            "SKU123456",
+            "GTIN1234567890123",
+            19.99,
+            true,
+            true,
+            false,
+            true,
+            true,
+            "Meta Title for SEO",
+            "meta, keywords, for, SEO",
+            "Meta description of the product for search engines.",
+            null,
+            List.of(789L, 101L, 112L),
+            new ArrayList<>(List.of(productVariationPostVm)),
+            new ArrayList<>(List.of(productOptionValuePostVm)),
+            List.of(234L, 567L),
+            89L
+        );
+
+        productVariationPutVm = new ProductVariationPutVm(
+            1L,
+            "Sample Product",
+            "sample-product",
+            "SKU12345",
+            "GTIN1234567890",
+            29.99,
+            101L,
+            new ArrayList<>(),
+            new HashMap<>()
+        );
+
+        productPutVm = new ProductPutVm(
+            "Sample Product Update",
+            "sample-product-update",
+            29.99,
+            true,
+            true,
+            false,
+            true,
+            true,
+            brand.getId(),
+            new ArrayList<>(),
+            "Short description of the product.",
+            "Detailed description of the product with features.",
+            "Specifications of the product.",
+            "SKU123456",
+            "GTIN1234567890123",
+            "Meta Title for SEO",
+            "Meta Keywords, Sample, Product",
+            "Meta Description of the product for search engines.",
+            10L,
+            new ArrayList<>(),
+            new ArrayList<>(List.of(productVariationPutVm)),
+            new ArrayList<>(),
+            new ArrayList<>(),
+            2L
+        );
+
+        productQuantityPostVm = new ProductQuantityPostVm(productOne.getId(), 1000L);
+        productQuantityPutVm = new ProductQuantityPutVm(productOne.getId(), 500L);
+    }
+
+    @AfterEach
+    public void clearTestData() {
+        productRelatedRepository.deleteAll();
+        productOptionRepository.deleteAll();
+        productImageRepository.deleteAll();
+        productOptionRepository.deleteAll();
+        productCategoryRepository.deleteAll();
+        categoryRepository.deleteAll();
+        productRepository.deleteAll();
+        brandRepository.deleteAll();
+    }
+
+    @Test
+    void test_getProductListBackoffice_shouldReturnProductList() {
+        given(getRequestSpecification())
+            .auth().oauth2(getAccessToken("admin", "admin"))
+            .when()
+            .get(PRODUCT_BACKOFFICE_URL)
+            .then()
+            .statusCode(HttpStatus.OK.value())
+            .body("productContent", hasSize(2))
+            .log().ifValidationFails();
+    }
+
+    @Test
+    void test_getProductFeature_shouldReturnProductList() {
+        given(getRequestSpecification())
```

Comment:
```
getRequestSpecification())
            .auth().oauth2(getAccessToken("admin", "admin"))
```
Extract this code into a method to avoid repetition code